rawx/chunk_info.go

```diff
@@ -226,10 +226,11 @@ func (chunk *chunkInfo) loadAttr(inChunk fileReader, chunkID string,
 		}
 		*(hs.ptr) = value
 	}
-
-	chunk.size, err = strconv.ParseInt(chunk.ChunkSize, 10, 63)
-	if err != nil {
-		return err
+	if chunk.ChunkSize != "" {
```

Comment:
A warning is raised in log but if attribute is missing, the GET of chunk will return 0 bytes without error.
IMO a return `fmt.Errorf` should be added in this case